xdc-plugin/xdc.cc

```diff
@@ -126,7 +130,28 @@ struct SetProperty : public Pass {
         if (design->top_module() == nullptr) {
             log_cmd_error("No top module detected\n");
         }
+        if (args.at(1) == "-dict") {
+            std::string dict_args = args.at(2).c_str();
```

Comment:
I believe that `c_str()` is obsolete here.

Reply:
Done, removed.